commercetools-models/src/main/java/io/sphere/sdk/orders/Order.java

```diff
@@ -284,6 +284,7 @@ public String toString() {
     List<Address> getItemShippingAddresses();
 
     @Nullable
+    @HasUpdateAction(value = "setStore", fields = {@PropertySpec(name = "store",type = Store.class)})
```

Comment:
The store property is a ResourceIdentifier<Store>

So this annotation will most likely not work. You will most likely add the update action manually. A test for this would also be good.